homeassistant/components/proximity/__init__.py

```diff
@@ -75,7 +79,7 @@ def setup_proximity_component(hass, name, config):
     )
     zone_id = f"zone.{config.get(CONF_ZONE)}"
 
-    proximity = Proximity(
+    proximity = Proximity(  # type:ignore[no-untyped-call]
```

Comment:
I'm really not sure how to add typing to the `Proximity` class.
I don't think it should block this PR.